Riot/Assets/en.lproj/Vector.strings

```diff
@@ -710,6 +710,7 @@ Tap the + to start adding people.";
 "room_details_topic" = "Topic";
 "room_details_favourite_tag" = "Favourite";
 "room_details_low_priority_tag" = "Low priority";
+"room_details_notifs" = "Notifications";
```

Comment:
This string exists in SDK, should I copy over the translations for the other languages?